docs/components/connectors/use-connectors/index.md

```diff
@@ -165,7 +165,11 @@ indicate internal website errors, which is why the website team is informed.
 
 The **Error Expression** property requires a [FEEL](/components/modeler/feel/what-is-feel.md) expression that yields a BPMN error object in the end. The BPMN error object can be an empty [context](/components/modeler/feel/language-guide/feel-data-types.md#context),
 [null](/components/modeler/feel/language-guide/feel-data-types.md#null), or a context containing at least a non-empty `errorType` and a non-empty `code` if the error type is `bpmnError`. You can use all available functionality provided by FEEL to produce this result.
-Use the provided FEEL function [`bpmnError`](#function-bpmnerror) to conveniently create a BPMN error object and the provided FEEL function [`jobError`](#function-jobError) to conveniently create a fail job object.
+
+Use the provided FEEL functions:
+
+- [`bpmnError`](#function-bpmnerror) to create a BPMN error object
+- [`jobError`](#function-jobError) to create a fail job object. This will result in a retry if retries >= 1, otherwise it will result in an incident.
```

Comment:
Proposal. Rephrase to "This will trigger a the [standard error handling behavior](link to docs) for failed jobs: The engine will retry the activity. Once retries are exhausted it will create an incident."

Reply:
@nikku thank you, this is great!

I found a doc (which I now link to in the latest commit) that explains the retry/incident logic in (more) plain English. I didn't want to re-write that doc here, since this Connectors page already covers a lot.

Let me know what you think of the latest commit!

Reply:
Works really well, the link (I think).